src/components/AddList.jsx

```diff
@@ -0,0 +1,49 @@
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import { createList } from '../api';
+import { useAuth } from '../api/useAuth.jsx';
+
+export default function AddList({ setListPath }) {
+	const [listName, setListName] = useState('');
+	const [message, setMessage] = useState('');
+	const navigate = useNavigate();
+	const { user } = useAuth();
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+		try {
+			const newList = createList(user.uid, user.email, listName);
```

Comment:
Not blocking: you could check if a user is present (aka if they're logged in or not) and throw up an alert prompting the user to login to create a new list if they aren't logged in in this handleSubmit function.

Reply:
Good idea :) I have a question though, what does it mean not blocking?

Also, we had casually discussed adding functionality later--- like private vs public routes or validation of some kind. If we do that, would adding this to the handleSubmit function be duplicitous?

Reply:
Update: we chatted about this via DM. Feel free to ignore that comment for now and merge when ready, I missed the convo about adding this later.